betty/serde/format/__init__.py

```diff
@@ -63,6 +64,10 @@ def dump(self, dump: Voidable[Dump]) -> str:
         """
 
 
+async def _format_app_repository() -> PluginRepository[FormatDefinition]:
+    return FORMAT_REPOSITORY
```

Comment:
Make this an entry point plugin repo again and add back the serde format plugin type documentation we removed some time ago.